### PR TITLE
refind: 0.12.0 -> 0.13.0

### PR DESCRIPTION
**Pull request by samueldr**
NEWS:

 - https://sourceforge.net/p/refind/code/ci/a1196dce2961aebbe35dc8bd56650119d9c4b786/tree/NEWS.txt#l1

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113328` at a938807b run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/ceb7082d41c7587769f4a784e0576999">refind</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/tools/bootloaders/refind/default.nix:26:5:
```
   |
26 |     ./0001-toolchain.patch
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/bootloaders/refind/default.nix:45:3:
```
   |
45 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/bootloaders/refind/default.nix [hunk 1]
```diff
@@ -14,17 +14,17 @@ in
 
 stdenv.mkDerivation rec {
   pname = "refind";
-  version = "0.12.0";
+  version = "0.13.0";
   srcName = "refind-src-${version}";
```
Comment (SuperSandro2000) [inline pkgs/tools/bootloaders/refind/default.nix:18]:
```suggestion
```

File: pkgs/tools/bootloaders/refind/default.nix [hunk 2]
```diff
@@ -14,17 +14,17 @@ in
...
 
   src = fetchurl {
     url = "mirror://sourceforge/project/refind/${version}/${srcName}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/tools/bootloaders/refind/default.nix:21]:
```suggestion
    url = "mirror://sourceforge/project/refind/${version}/${srcName}.tar.gz";
```
```suggestion
    url = "mirror://sourceforge/project/refind/${version}/refind-src-${version}".tar.gz";
```

**Comment by samueldr** [comment]
@AndersonTorres any reason the [last commit](https://github.com/NixOS/nixpkgs/pull/113328/commits/a68097f579d245e21ba67f77326adacd1d63bcb5), changing from your original contribution, would be undesirable?

**Comment by AndersonTorres** [comment]
@samueldr no problem to me.

**Pull request merged**